flexmeasures/data/scripts/grid_weather.py

```diff
@@ -375,6 +383,13 @@ def save_forecasts_in_db(
                             flexmeasures_sensor_type, lat=location[0], lng=location[1]
                         )
                         if weather_sensor is not None:
+                            # Complain if the nearest weather sensor is further away than 1 degree
```

Comment:
Note that the effect of this logic depends strongly on latitude. For example, at high latitudes, 1 degree difference in longitude may be just a few km.

As an intermediate remedy (no need for a config variable at this point, right?), I suggest to at least make the 1 degree a function argument so it will appear in a docstring (including my note above).